hack/Makefile

```diff
@@ -0,0 +1,52 @@
+lint-swagger:
+	spectral lint swagger.yaml
+
+generate-go-client:
+	rm -rf client/ models/*.go
+	goswagger generate client --template=stratoscale -f swagger.yaml
+
+generate-go-server:
+	rm -rf restapi/
+	goswagger generate server --template=stratoscale -f swagger.yaml
+
+generate-python-client:
+	./hack/generate.sh generate_python_client
+
+generate-mocks:
+	find . -name 'mock_*.go' -type f -delete
+	go generate -v $(shell go list ./... | grep -v 'assisted-service/models\|assisted-service/client\|assisted-service/restapi')
+
+generate-migrations:
+	go run ./tools/migration_generator/migration_generator.go -name=${MIGRATION_NAME}
+
+generate-keys: ${BUILD_FOLDER}
+	(cd ./tools && go run auth_keys_generator.go -keys-dir=${BUILD_FOLDER})
+
+generate-events:
+	rm -rf internal/common/events
+	mkdir -p internal/common/events
+	tools/generate_events.py ./docs/events.yaml internal/common/events/events.go
+
+remove-dashes-and-dots:
```

Comment:
Maybe we can make it private target (`--remove-dashes-and-dots`)?

Reply:
sure why not